test/scripts/test.sh

```diff
@@ -1,122 +1,148 @@
 #!/bin/bash
 
-#set -e
-set -o pipefail
+set -e
+#set -o pipefail
```

Comment:
With this option on, it never gets executed

Reply:
To understand why this setting is needed you can add the following line to the file `01_period_tbl.sql` as first test
```
SELECT COUNT(*) FROM tbl_period;
```
When `set -e` the difference in the output is not captured. When `set -o pipefail` it is
```
esteban@DESKTOP-9FBND60:~/src/PR/MobilityDB/build$ make test
Running tests...
Test project /home/esteban/src/PR/MobilityDB/build
        Start   1: build
  1/111 Test   #1: build .............................   Passed    0.24 sec
        Start   2: setup
  2/111 Test   #2: setup .............................   Passed    2.92 sec
        Start   3: create_extension
  3/111 Test   #3: create_extension ..................   Passed   17.10 sec
        Start   5: load_tables
  4/111 Test   #5: load_tables .......................   Passed    3.60 sec
        Start   6: 01_period
  5/111 Test   #6: 01_period .........................   Passed    0.05 sec
        Start   7: 01_period_hash_extended_pg11
  6/111 Test   #7: 01_period_hash_extended_pg11 ......   Passed    0.03 sec
        Start   8: 01_period_tbl
  7/111 Test   #8: 01_period_tbl .....................***Failed    0.12 sec
        Start   9: 03_timestampset
  8/111 Test   #9: 03_timestampset ...................   Passed    0.04 sec
[...]
```
For exploring all your changes to the test.sh file please keep this modified version of the test so you can detect the errors.

Reply:
I will not modify the tests